apps/gitness/src/pages/pipeline-list.tsx

```diff
@@ -13,9 +13,25 @@ import {
   Spacer,
   Text
 } from '@harnessio/canary'
-import { useListPipelinesQuery, TypesPipeline, ListPipelinesOkResponse } from '@harnessio/code-service-client'
+import {
+  useListPipelinesQuery,
+  TypesPipeline,
+  ListPipelinesOkResponse,
+  useMembershipSpacesQuery
+} from '@harnessio/code-service-client'
 import { PipelineList, MeterState, TopBarWidget, PaddingListLayout, SkeletonList } from '@harnessio/playground'
 import { ExecutionState } from '../types'
+import { Link } from 'react-router-dom'
+
+interface Project {
+  id: string | undefined
+  identifier: string | undefined
+}
+
+interface Project {
+  id: string | undefined
+  identifier: string | undefined
+}
```

Comment:
duplicate interface.. how did typescript and eslint not catch this?

also, why do we have a hand-coded interface here? all API interfaces should be coming from the service client

Reply:
Got it! I will check on that if they have related interface there